src/test/java/uk/gov/hmcts/reform/sscs/service/scheduler/CcdActionExecutorTest.java

```diff
@@ -44,7 +44,7 @@ public void setup() {
 
         caseDetails = SscsCaseDetails.builder().caseTypeId("123").build();
 
-        newSscsCaseData = SscsCaseData.builder().build();
+        SscsCaseData newSscsCaseData = SscsCaseData.builder().build();
```

Comment:
this object is defined as a filed already. Please, remove this local variable. Thanks

Reply:
Done